src/imports.jl

```diff
@@ -0,0 +1,41 @@
+# Check that packages like JSON3 are already imported by the user.
+
+function require_import(m::Symbol)
+    pkgid = Base.identify_package(string(m))
```

Comment:
Heads up, I believe I made a big mistake here:

Using `identify_package` might break precompilation: https://github.com/JuliaLang/julia/pull/45287/files#r873871094

Reply:
Exactly, as least that's how ImageIO did in the old versions.